### ISSUE TITLE
Todos

### ISSUE THREAD

**Issue by aidangomez** [bryanlimy/mnist-classification-with-tensorflow#1]
Very well done Bryan! Nice attention to detail and code design, I'm impressed.

Todo:
- [ ] in /data/ you should have an "mnist" file and a "seq2seq" for each of the tasks
- [ ] LSTM is not done, please talk to rest of the team and collaborate with them to complete this
- [ ] AE: shouldn't specify each layer, just have a loop and have a 'depth' hyperparam
- [ ] AE: perhaps more dropout? like at each layer
- [ ] MLP: a single layer is a bit shallow, add some level of depth
- [ ] add some text files that report performance under ablations of depth, dropout etc
